recipes/libnuma/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+class LibnumaConan(ConanFile):
+    name = "libnuma"
+    description = "NUMA support for Linux."
+    license = "GPL-2.0-or-later"
```

Comment:
https://github.com/numactl/numactl#license-copyrights-acknowledgements

Looking this over I see there is only the `libnuma` that's packaged which is LGPL...

Does this make sense

```suggestion
    license = "LGPL-2.1-or-later"
```

Reply:
There are two licenses in the repository. I missed the fact in README that libnuma is a subject for less strict LGPL.  Thanks for pointing out!